frontend/stepper/js/BlocklyEditor.tsx

```diff
@@ -38,6 +42,17 @@ export const BlocklyEditor = (props: BlocklyEditorProps) => {
         log.getLogger('editor').debug('imported content', context.blocklyHelper.programs[0].blockly);
         context.blocklyHelper.reloading = true;
         context.blocklyHelper.loadPrograms();
+
+        // Check that all blocks exist and program is valid. Otherwide, reload default answer and cancel
+        try {
+            context.blocklyHelper.programs[0].blocklyJS = context.blocklyHelper.getCode("javascript");
+        } catch (e) {
+            console.error(e);
```

Comment:
I haven't used the logger for the errors because most of the time, we want to see them and not hide them. Though you're right I could use the logger with a high priority for the errors. I leave this for another piece of work because that would need updating all the `console.error` to be done properly.